Activities/Python/UiPath.Python.Activities/LoadScript.cs

```diff
@@ -45,50 +45,48 @@ protected async override Task<Action<AsyncCodeActivityContext>> ExecuteAsync(Asy
 #if ENABLE_DEFAULT_TELEMETRY
             telemetryOperation = RuntimeTelemetryService.CreateExecutionOperation(this, context);
 #endif
+            try
+            {
+                IEngine pythonEngine = PythonScope.GetPythonEngine(context);
 
-            IEngine pythonEngine = PythonScope.GetPythonEngine(context);
+                string scriptFile = ScriptFile.Get(context);
+                string scriptCode = Code.Get(context);
 
-            string scriptFile = ScriptFile.Get(context);
-            string scriptCode = Code.Get(context);
+                // safeguard checks
+                if (scriptFile.IsNullOrEmpty() && scriptCode.IsNullOrEmpty())
```

Comment:
```suggestion
                if (scriptFile.IsNullOrWhiteSpace() && scriptCode.IsNullOrWhiteSpace())
```

We also seem to check this twice for each, maybe if we need to reuse this value, simply store it in a var?